worker/tasks.py

```diff
@@ -614,9 +627,9 @@ def docker_alpine_run(ctx):
     """
     Run a container of the Alpine Docker image created in the docker_alpine task
     """
-    with ctx.cd(f'"{WORKER_DIR}"'):
+    with cd_worker():
         ctx.run(
-            f'"{DOCKER}" run --name=mediasoupDockerAlpine -it --rm --privileged --cap-add SYS_PTRACE -v "{WORKER_DIR}/../:/mediasoup" mediasoup/docker-alpine:latest',
+            f'"{DOCKER}" run --name=mediasoupDockerAlpine -it --rm --privileged --cap-add SYS_PTRACE -v "{WORKER_DIR}/../:/foo bar/mediasoup" mediasoup/docker-alpine:latest',
```

Comment:
BTW these tasks with container image builds do not really belong to this file, I think they (along with `Dockerfile`s should be moved to CI).

Reply:
If it is not used in CI then it should be. This is a test case for certain behavior of `tasks.py`. The fact that it is orchestrated by `tasks.py` itself is kind of an accident.

In fact it seems to be orchestrated in a strange way by running command inside container with `mediasoup/docker-alpine:latest` image that modifies local files, rather than using a generic image and trying to build a new container using `docker build`. Moreover, it modifies local files due to `-v` rather than being ephemeral, so you'll have to clean up `node_modules` if you want to run something after than natively on macOS or Windows because the files under `node_modules` are now compiled for a completely different OS.

All this is of course unrelated to this PR, just something I noticed.

Reply:
It behaves that way on purpose. For local development purposes I need to switch to Linux (Docker) and when I do that I want to be able to edit mediasoup source files directly on their original path. Yes, I need to uninstall deps and subprojects and everything before switching between macOS and Linux/Docker, that's the price to pay.